db/migrate/20240220222611_create_merchants.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMerchants < ActiveRecord::Migration[7.1]
+  def change
+    create_table :merchants do |t|
+      t.string :name
+
```

Comment:
Just looking through some of the user stories it may be easier for us to create a relationship between merchants and invoices versus having to join via ActiveRecord